### PR TITLE
Check for whitespace in CSV sample file

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
DR has noticed that some fields in sample files should be blank but actually contain whitespace which could cause issues further down the line if recognised as a value

# What has changed
Added in a new validator to check if field starts with a whitespace
Created unit test for happy and unhappy path

# How to test?
Edit a test sample in the repo by adding in blank spaces where the field is expected to be blank
Then run `pipenv run python validate_sample.py <sample_file>`
Validation error should come up specifying which row/column

# Links
https://trello.com/c/yUU8RM7J/830-add-white-space-check-to-sample-load-validator-3

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Add white space check to sample load validator (3)](https://trello.com/c/yUU8RM7J/830-add-white-space-check-to-sample-load-validator-3)

File: validators.py [hunk 1]
```diff
@@ -80,3 +80,11 @@ def validate(value: Iterable) -> None:
                            f'unexpected values: {value_as_set.difference(expected_set)}'))
 
     return validate
+
+
+def no_whitespace_check():
+    def validate(value):
+        if value.startswith(" "):
```
Comment (NickGrantX) [inline validators.py]: I'm not sure if `startsWith` qualifies as "contains whitespace". I think we should compare the original with the `.strip()` result if you want to know whether there was whitespace at the beginning or the end of the value.

You can use `value.replace(" ", "") == ''` to check to see if a value is _entirely_ whitespace.

**Review by NickGrantX — CHANGES_REQUESTED** [review]
See comments

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by hbrace — APPROVED** [review]

**Pull request merged**